config/settings.yml

```diff
@@ -930,6 +930,9 @@
     - MoveIntoResourcePool
     :VmSuspendedEvent:
     - SuspendVM_Task
+:excon:
+  :openstack_omit_default_port: true
+  :openstack_read_timeout: 60
```

Comment:
Can this `excon` section be moved to the `:ems` section?

So, it would look something like:

```yaml
:ems
  :ems_openstack
    :excon
      :omit_default_port: true
      :read_timeout: 60
```

Then, accessing looks like:

```ruby
extra_options[:omit_default_port] = ::Settings.ems.ems_openstack.excon.omit_default_port
```

The nesting under `excon` is up to you.  As long as the settings are contained under `ems_openstack`.  Then, when the provider is extracted, these settings are easier to track down.